app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -44,13 +44,28 @@ public function index(Request $request)
             ->where('status', '=', 'approved')
             ->select('posts.id as id', 'signups.campaign_id as campaign_id', 'posts.status as status', 'posts.caption as caption', 'posts.url as url', 'posts.created_at as created_at', 'posts.signup_id as signup_id');
 
-        // Only return Posts for the given campaign_id (if there is one)
         if (isset($request->filter)) {
+            // Only return Posts for the given campaign_id (if there is one)
             if (array_key_exists('campaign_id', $request->filter)) {
                 $campaign_id = $request->filter['campaign_id'];
 
                 $query = $query->where('campaign_id', '=', $campaign_id);
             }
+            // Exclude Posts if exclude param is present
```

Comment:
We want to be able to exclude posts from the response when we're populating the gallery on the Javascript side when we click the "view more" button. We don't want to show the first 6 images that are already displayed so we'll just exclude those!